listenbrainz/webserver/views/api.py

```diff
@@ -722,6 +722,7 @@ def get_tags_dataset():
     tag = request.args.getlist("tag")
     if tag is None:
         raise APIBadRequest("tag param is missing")
+    tag = [t.lower() for t in tag]
```

Comment:
I am not sure if all tag names in the database are lower cased. I'll check if that is the case, then the fix would work. If not, a case insensitive match should be performed in the SQL query. Will update.

Reply:
Ping @amCap1712

Reply:
It appears that all tags are in fact lowercase:

```
musicbrainz_db=# select count(distinct(lower(name))) from tag;
 count  
--------
 213811
(1 row)

musicbrainz_db=# select count(*) from tag;
 count  
--------
 213811
(1 row)
```